Automattic/wp-calypso
# Backups needs to go to `cloud.jetpack.com` from the command palette on Atomic

Close https://github.com/Automattic/dotcom-forge/issues/5822

## Proposed Changes

This PR updated the link of Backup in the command palette so that it redirects to `cloud.jetpack.com/backup/<site>`

## Testing Instructions



* Prepare an Atomic site
* Open the command palette with Cmd+K 
* Select `Open Jetpack Backup`
	* <img width="1429" alt="Screenshot 2024-02-29 at 16 55 53" src="https://github.com/Automattic/wp-calypso/assets/5287479/1350835a-e022-488d-960c-a40d19d2a9e7">
* See it redirects to `cloud.jetpack.com/backup/<site>`

---
* Prepare a Simple site
* Open the command palette with Cmd+K 
* Select `Open Jetpack Backup`
* See it redirects to `wordpress.com/backup/<site>`


## Pre-merge Checklist
